README.md

```diff
@@ -1,134 +1,94 @@
-# Fastly Controller
+# fastly-controller
```

Comment:
I think we're deleting a tiny bit too much here 🙈

Reply:
oops yeah :)